sdk/cmake/host-tools.cmake

```diff
@@ -26,6 +26,48 @@ function(setup_host_tools)
         set(HOST_MODULE_SUFFIX ".so")
     endif()
 
+    # Normalize to the same format as our own ARCH, and add one for the VC shell
+    string(TOLOWER "${CMAKE_HOST_SYSTEM_PROCESSOR}" lowercase_CMAKE_HOST_SYSTEM_PROCESSOR)
+    if(lowercase_CMAKE_HOST_SYSTEM_PROCESSOR STREQUAL x86_64 OR lowercase_CMAKE_HOST_SYSTEM_PROCESSOR STREQUAL amd64)
```

Comment:
This stuff is for Windows only, and there `CMAKE_HOST_SYSTEM_PROCESSOR` is set with `PROCESSOR_ARCHITECTURE` value, which itself is already compatible with what you need for vcvarsall.bat.
Maybe this is not needed at all?

Reply:
Ah, you need to do a comparison with `ARCH`...